Instagram/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,68 @@
+#import "LoginViewController.h"
+#import "Parse/Parse.h"
+
+@interface LoginViewController ()
+
+@property (strong, nonatomic) IBOutlet UITextField *usernameField;
+@property (strong, nonatomic) IBOutlet UITextField *passwordField;
+- (IBAction)didTapLogin:(UIButton *)sender;
```

Comment:
nit: It's common to put a newline between properties, class, and instance methods. Usually it looks like:

```
@interface Foo ()

@property prop1
@property prop2

+ (void)classMethod1
+ (void)classMethod2

- (void)instancemethod1
- (void)instancemethod2

@end
```